libp2p/__init__.py

```diff
@@ -93,8 +98,14 @@ def initialize_default_swarm(
     # TODO TransportUpgrader is not doing anything really
     # TODO parse muxer and sec to pass into TransportUpgrader
     muxer = muxer_opt or ["mplex/6.7.0"]
-    sec = sec_opt or {TProtocol("insecure/1.0.0"): InsecureTransport("insecure")}
-    upgrader = TransportUpgrader(sec, muxer)
+    private_key_bytes = private_key.export_key("DER")
```

Comment:
Just a note: I learned we lack `KeyBook` implementation, which is useful when we need to store peers' public keys in the future. Ref: https://github.com/libp2p/go-libp2p-peerstore/blob/master/pstoremem/keybook.go

Reply:
yeah let's make an issue